draft-scenarios/campaigns/castle_greek_gift.tex

```diff
@@ -0,0 +1,232 @@
+% !TeX spellcheck = en_US
+\cleardoublepage\phantomsection\addcontentsline{toc}{section}{\protect\numberline{} {} {} {} {}The Queen's Gambit}
+\addscenariosection[subsection]{1}{Castle Campaign $-$ The Queen's Gambit}{1. Greek Gift}{\images/catherine.png}
+
+\begin{multicols*}{2}
+
+\textbf{Author:} Tyson Heckert
+
+\textbf{Source:} \href{https://travelledtales.com}{Travelled Tales}
+
+\textit{Steadwick is free, but that doesn't mean it is safe.
+Dark powers still roam Erathia, and one now moves its pieces into position.
+While Catherine grapples with her responsibility—and her demons—she must be a bulwark against evil.}
+
+\subsection*{\MakeUppercase{Scenario length}}
+
+This Scenario plays out over 8 Rounds.
+
+\subsection*{\MakeUppercase{Player setup}}
+
+\textbf{Faction:} Castle
+
+\textbf{Faction Hero:} Any Castle Hero. If you are not Catherine, you are in her entourage for this story.
+
+\textbf{Starting Resources:} 10 \svg{gold}, 3 \svg{building_materials}, 1 \svg{valuables}
+
+\textbf{Starting Income:} 10 \svg{gold}, 0 \svg{building_materials}, 0 \svg{valuables}
+
+\textbf{Starting Units:}
+\begin{itemize}
+  \item A Few Halberdiers, A Few Marksmen
+\end{itemize}
+
+\textbf{Town Buildings:} \svgunit{bronze} Dwelling, Citadel
+
+\subsection*{\MakeUppercase{AI Hero setup}}
+
+\textbf{Faction:} Necropolis
+
+\textbf{Enemy Army:} A Pack of Skeletons, A Pack of Zombies, A Few Dread Knights, A Few Ghost Dragons
+
+\textbf{Enemy Deck:} 4 x Might cards
+
+\textbf{Special:} Place the Cloak of the Undead King IV and VI cards aside
+
+\subsection*{\MakeUppercase{Map setup}}
+
+Take the following Map Tiles and arrange them as shown in the Scenario map layout:
+
+\textbf{1 × Starting (I) Map Tile}
+\begin{itemize}
+    \item 1 x Castle (S3)
+\end{itemize}
+
+\textbf{3 × Far (II--III) Map Tile}
+\begin{itemize}
+    \item 3 x Castle (F3, F6, F9)
+\end{itemize}
+
+\textbf{2 × Near (IV--V) Map Tile}
+\begin{itemize}
+    \item 2 x Castle (N3, N6)
+\end{itemize}
+
+\subsection*{\MakeUppercase{Heroes placement}}
+
+Place your Castle Hero on the center Field of the S3 Castle starting map Tile.
+
+Place a Necropolis Hero on the center Field of the N3 Castle Tile.
+
+\subsection*{\MakeUppercase{Victory Conditions}}
+
+Defeat the enemy army.
+
+\subsection*{\MakeUppercase{Defeat Conditions}}
+\begin{itemize}
+  \item You lose one combat encounter
+  \item You run out of time at the end of the 8th Round
+\end{itemize}
+\end{multicols*}
+
+\newpage
+
+\begin{multicols}{2}
+
+\subsection*{\MakeUppercase{Timed Events}}
+
+\textbf{\nth{1} Round:}
+\begin{itemize}
+  \item Read "Distant Threat" section
```

Comment:
```suggestion
  \item Read ``Distant Threat'' section
```
Please replace all the quotes like this. LaTeX doesn't like plain `"`. The former is rendered properly according to English typography.